### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by MLnick**

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def test_predict():
 
     formats = ['JPEG', 'PNG']
     model_endpoint = 'http://localhost:5000/model/predict'
-    file_path = 'assets/cat.jpg'
+    file_path = 'samples//cat.jpg'
```
Comment (splovyt) [inline tests/test.py]: The `//` is probably unintentional here

File: tests/test.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def test_predict():
 def test_invalid_input():
 
     model_endpoint = 'http://localhost:5000/model/predict'
-    file_path = 'assets/README.md'
+    file_path = 'samples//README.md'
```
Comment (splovyt) [inline tests/test.py]: Same here with the `//`

**Comment by splovyt** [comment]
I have also changed the `assets` reference in the README to `samples`. 

Should we also add a license header to the `tests.py` file? I think we did that in the `MAX-Skeleton`.

Thanks Nick.

File: tests/test.py (hunk 1 above)
Comment (MLnick) [reply tests/test.py]: Yup, find-and-replace error!

**Pull request merged**